tests/Messages/BatchMessageTest.php

```diff
@@ -41,6 +41,20 @@ public function testAddRecipient()
         $this->assertEquals(1, $array['recipients']['to']);
     }
 
+    public function testAddRecipientWithoutFirstAndLastName()
+    {
+        $message = $this->client->BatchMessage($this->sampleDomain);
+        $message->addToRecipient('test@samples.mailgun.org', 'this-is-not-an-array');
+        $messageObj = $message->getMessage();
+        $this->assertEquals(['to' => ['test@samples.mailgun.org']], $messageObj);
+
+        $reflectionClass = new \ReflectionClass(get_class($message));
+        $property = $reflectionClass->getProperty('counters');
+        $property->setAccessible(true);
+        $array = $property->getValue($message);
```

Comment:
No no, I do not want to get rid of PHPUnit. Just these four lines of code: 

```php
$reflectionClass = new \ReflectionClass(get_class($message));
$property = $reflectionClass->getProperty('counters');
$property->setAccessible(true);
$array = $property->getValue($message);
```
We have written that on several places. I suggest to use NSA because we can achieve the same with: 

```php
$array = NSA::getProperty($message, 'counters');
```